### PR TITLE
[NO-TICKET] Doc site design updates

### PR DESCRIPTION
**Pull request by line47**
## Summary

Added some small design changes 

### Added

- Added some styling to the props and style tables to add color to the table header bg and lighten the row border color

### Changed

- Changed the props and style tables to be 100% width and be styled as the borderless style
- Changed the spacing in the page header to slim it down some 
- Changed the spacing in the left nav top to match the page header
- Changed the font sizing for page title H2, and H3 in page content 
- Changed spacing for the table of contents 

## How to test

View the demo URL https://cmsgov.github.io/design-system/branch/NO-TICKET-doc-site-design-updates

### CODE REVIEW

File: packages/docs/src/styles/pages/layout.scss [hunk 1]
```diff
@@ -27,6 +27,13 @@ body {
     :nth-child(2) {
       margin-top: 0;
     }
+    > h2 {
+      font-size: $font-size-3xl;
+      font-weight: normal;
+    }
+    > h3 {
+      font-size: $font-size-2xl;
+    }
```
Comment (pwolfert) [inline packages/docs/src/styles/pages/layout.scss:36]: We now have some h4's. Do they need to be specified here?

**Review by pwolfert — COMMENTED** [review]
I don't know what happened to my earlier review of this. I know I made comments on this earlier today, but they somehow got lost.

**Review by pwolfert — COMMENTED** [review]
The design tweaks look nice, by the way 👍

File: packages/docs/src/styles/pages/layout.scss (hunk 1 above)
Comment (line47) [reply packages/docs/src/styles/pages/layout.scss:36]: They probably should be - good callout!

**Review by pwolfert — APPROVED** [review]

**Pull request merged**